zenoh/__init__.py

```diff
@@ -17,3 +17,7 @@
     from . import ext
 except ImportError:
     pass
+
+
+def _unstable(item):
```

Comment:
This function is not needed in the code. It's fine to have private items in stubs that don't correspond to a real exported item.

Reply:
it is needed for stubs_check.py test to pass.

Reply:
Then modify `stubs_check.py`

Reply:
done